Wiredcraft/Section/Home/Controller/WCHomeViewController.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  WCHomeViewController.swift
+//  Wiredcraft
+//
+//  Created by codeLocker on 2020/6/10.
+//  Copyright © 2020 codeLocker. All rights reserved.
+//
+
+import UIKit
+import RxCocoa
+import RxSwift
+import RxDataSources
+import ESCategory_swift
+
+/*
+ Home
+*/
+
+class WCHomeViewController: WCBaseViewController {
+
+    /// search input view
+    fileprivate var searchBar: UISearchBar = {
```

Comment:
Why is this `fileprivate` instead if `private`?

Reply:
when write the code, i think I think I might use this widget in the extension of WCHomeViewController.
so i used fileprivate here.